manifests/init.pp

```diff
@@ -88,6 +88,7 @@
   $lingering_timeout                                         = '5s',
   Optional[Enum['on', 'off']] $etag                          = undef,
   Optional[String] $events_use                               = undef,
+  Optional[Array[Nginx::DebugConnection]] $debug_connection  = undef,
```

Comment:
Since it takes an array, maybe `$debug_connections` (plural) would be a better name.
(or maybe `$debug_connection_addresses`).

Ether way, instead of making it `Optional`, I think it'd be nicer to just default it to an empty array `[]`.

Reply:
right, thanks!